src/commands/chain.js

```diff
@@ -82,6 +85,19 @@ async function main(archwayd, name, options = {}) {
                 process.stdout.write(`${data}`);
             });	
             break;
+        case 'snapshot':
+            const rootPath = path.dirname(await Config.path(this.pathPrefix));
+            const nowDate = new Date();
+            console.log("Make a snapshot");
+            const mkDir = spawn("mkdir", [`${rootPath}/snapshots`]);
+            if (mkDir.error != null){
+                throw mkDir.error;
+            }
+            const snapshot = spawn("tar", ["zcvf", `${rootPath}/snapshots/chaindata_${nowDate.toISOString()}.tar.gz`,`${archwayd.archwaydHome}/data`, "-C", "~/.archway"]);
```

Comment:
"~/.archway" hardcoded here.
It should be retrieved from `archwayd` because home directory can be changed.

Reply:
Try to use `tar -C ~/.archway -zcvf test.tar.gz data`
This tested on OSX and Linux.